test/distrib/cpp/run_distrib_test_cmake.bat

```diff
@@ -64,7 +64,7 @@ mkdir cmake
 cd cmake
 mkdir build
 cd build
-cmake -DCMAKE_INSTALL_PREFIX=%INSTALL_DIR% -DOPENSSL_ROOT_DIR=%OPENSSL_DIR% -DOPENSSL_INCLUDE_DIR=%OPENSSL_DIR%/include ../.. || goto :error
+cmake -DCMAKE_INSTALL_PREFIX=%INSTALL_DIR% -DOPENSSL_ROOT_DIR=%OPENSSL_DIR% -DOPENSSL_INCLUDE_DIR=%OPENSSL_DIR%/include -DZLIB_LIBRARY=%INSTALL_DIR%/lib/zlibstatic.lib -DZLIB_INCLUDE_DIR=%INSTALL_DIR%/include ../.. || goto :error
```

Comment:
Nice find, thanks for the detailed description in the commit message. 
I realized I'm still unhappy with the way zlib gets included - ideally setting ZLIB_ROOT should be enough here.
(as per https://cmake.org/cmake/help/v3.0/module/FindZLIB.html, where only ZLIB_ROOT is an allowed hint).
I'll see what the tests say.

Reply:
I added a commit with some cleanup and it seems that the tests are passing.